app/controllers/concerns/metadata.rb

```diff
@@ -76,7 +76,9 @@ def last_offset(total)
     private
 
     def base_link_url
-      "#{path_prefix}/#{Settings.app_name}/#{controller_name}"
+      api_version = request.fullpath.delete_prefix("#{path_prefix}/#{Settings.app_name}")
+      api_version.delete_suffix!("/#{controller_name}")
```

Comment:
What if api version is not set at all? Like when you're trying to access the API just via `/api/compliance/...`

Reply:
Than `api_version` should endup being an empty string